src/actions.ts

```diff
@@ -290,69 +302,61 @@ export const blur = () => ({
 
 // Key Bindings
 
-type KeyDownHandlers<Cell: Types.CellBase> = {
-  [eventType: string]: KeyDownHandler<Cell>
+type KeyDownHandlers<Cell> = {
+  [eventType: string]: KeyDownHandler<Cell>;
 };
 
 /** @todo handle inactive state? */
-const keyDownHandlers: KeyDownHandlers<*> = {
+const keyDownHandlers: KeyDownHandlers<any> = {
   ArrowUp: go(-1, 0),
   ArrowDown: go(+1, 0),
   ArrowLeft: go(0, -1),
   ArrowRight: go(0, +1),
   Tab: go(0, +1),
-  Enter: edit,
+  Enter: edit as KeyDownHandler<any>,
   Backspace: clear,
   Escape: blur
 };
 
-const editKeyDownHandlers: KeyDownHandlers<*> = {
-  Escape: view,
+const editKeyDownHandlers: KeyDownHandlers<any> = {
+  Escape: view as KeyDownHandler<any>,
   Tab: keyDownHandlers.Tab,
   Enter: keyDownHandlers.ArrowDown
 };
 
-const shiftKeyDownHandlers: KeyDownHandlers<*> = {
+const shiftKeyDownHandlers: KeyDownHandlers<any> = {
   ArrowUp: modifyEdge("row", -1),
   ArrowDown: modifyEdge("row", 1),
   ArrowLeft: modifyEdge("column", -1),
   ArrowRight: modifyEdge("column", 1)
 };
 
-const shiftMetaKeyDownHandlers: KeyDownHandlers<*> = {};
-const metaKeyDownHandlers: KeyDownHandlers<*> = {};
+const shiftMetaKeyDownHandlers: KeyDownHandlers<any> = {};
+const metaKeyDownHandlers: KeyDownHandlers<any> = {};
 
-function getActive<Cell: Types.CellBase>(state: Types.StoreState<Cell>): ?Cell {
+function getActive<Cell>(state: Types.IStoreState<Cell>): any {
```

Comment:
```suggestion
function getActive<Cell extends Types.CellBase>(state: Types.IStoreState<Cell>): any {
```

Reply:
Pretty much everywhere you removed the flow constraint you should replace with extends